requests/packages/urllib3/connection.py

```diff
@@ -6,18 +8,13 @@
 from .packages import six
 
 try:  # Python 3
-    from http.client import HTTPConnection as _HTTPConnection, HTTPException
+    from http.client import HTTPConnection as _HTTPConnection
```

Comment:
Ugh, I need to remember to go clean all of these imports up in urllib3 because some of the super shitty "future" modules that "backport" Python 3 module names to Python 2 will break things for us here and for urllib3 in general.

There's nothing wrong with 2.9.0, this is just something that I know will bite us.